cmd/nix/build.go

```diff
@@ -0,0 +1,70 @@
+package nix
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func Build(rootDirectory, target string, showTrace bool) (err error) {
+	if err != nil {
+		errorMessage := fmt.Sprintf(
+			"Error looking for root directory %s", err.Error(),
+		)
+
+		return errors.New(errorMessage)
+	}
+
+	currentDirectory, err := os.Getwd()
+	if err != nil {
+		log.Println(err)
+	}
+
+	path, attr, err := TargetToAttr(rootDirectory, currentDirectory, target)
+	if err != nil {
+		errorMessage := fmt.Sprintf(
+			"Error mapping target to directory %s", err.Error(),
+		)
+
+		return errors.New(errorMessage)
+	}
+
+	pathToBuild := strings.Trim(fmt.Sprintf("%s/%s", path, attr), "/")
+	pathToBuild = strings.Replace(pathToBuild, "/", ".", -1)
+
+	args := []string{
+		"build",
+		"--include", fmt.Sprintf("prj_root=%s", rootDirectory),
+		"--json",
+		"-f", "<prj_root>",
+		pathToBuild,
```

Comment:
Is it possible to keep the same arguments as https://github.com/numtide/bld/blob/fd54753c36dc0253a3f5b6a61e630410a6601695/bld.sh#L52-L78 ?